cmd/state-installer/test/integration/installer_int_test.go

```diff
@@ -137,6 +141,12 @@ func (suite *InstallerIntegrationTestSuite) AssertConfig(ts *e2e.Session) {
 	}
 }
 
+func (s *InstallerIntegrationTestSuite) setupPayload(ts *e2e.Session) {
```

Comment:
Now `ts.Dirs.InstallerPayload` is part of the testsuite, but is only used here. I think we can move this method to the test suite (call it `SetupInstallerPayload()` or something), or alternatively move `ts.Dirs.InstallerPayload` outside of the test suite if it has no ownership of it.